src/app/components/BibPage/BottomBibDetails.jsx

```diff
@@ -20,6 +20,7 @@ const BottomBibDetails = ({ bib, resources }) => {
     { label: 'Publication Date', value: 'serialPublicationDates' },
     { label: 'Description', value: 'extent' },
     { label: 'Donor/Sponsor', value: 'donor' },
+    { label: 'Dimensions', value: 'dimensions'},
```

Comment:
While this is perfectly fine and worthy of approval (such fanciness 🥇) I'm curious if we want to put Dimensions in this position if the array.

The list is not in alphabetical order, and it doesn't really seem to be in any particular order, so should it be placed in the 6th position, a different position, or does it not matter?

Reply:
i asked sean and he said at the end.

Reply:
ok